config.rb

```diff
@@ -65,7 +64,7 @@
 # Fetch via Contentful
 # NOTE: This must exist in this file, cannot be an extension as `middleman-contentful` doesn't appear to be able to initiate from any source other than config.rb.
 # Do note, you MUST run `middleman contentful` prior to `middleman [build|server]` as contentful does not work properly with lifecycle hooks and data will be parsed post-configuration, pre-build (where other things can't hook into it)
-[ ContentfulConfig::PartnersSpace, ContentfulConfig::LandingPagesSpace ].each do |space|
+[ ContentfulConfig::PartnersSpace ].each do |space|
```

Comment:
Almost there!  This is just there for the index right?

Reply:
yup!